dev/new-rule/index.js

```diff
@@ -1,119 +1,10 @@
-// eslint-disable-next-line eslint-comments/disable-enable-pair
-/* eslint-disable no-console */
+const { createEnv } = require('yeoman-environment');
 
-const fs = require('fs');
-const path = require('path');
+// eslint-disable-next-line wrap-iife
```

Comment:
What is `wrap-iife`?

Can you write it as:

```js
async function main() {
  let env = createEnv();

  env.register(require.resolve('./new-rule-generator'), 'ember-template-lint:new-rule');

  await env.run('ember-template-lint:new-rule');
}

main();
```

This seems fine (old habbits with a "main function" die hard I guess), and avoids the linting comment

Reply:
Yep, makes sense.